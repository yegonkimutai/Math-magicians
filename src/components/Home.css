.home-main {
    margin: 10%;
    text-align: center;
    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
}