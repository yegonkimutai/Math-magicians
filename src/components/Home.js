import './Home.css';

const Home = () => {
    return (
        <div className="home-main">
            <h1>Welcome to Calculator App!</h1>
            <h2>Let's Do some Math!</h2>
        </div>
    )
}

export default Home