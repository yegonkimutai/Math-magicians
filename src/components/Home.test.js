import renderer from 'react-test-renderer';
import Home from './Home';

it('renders', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
