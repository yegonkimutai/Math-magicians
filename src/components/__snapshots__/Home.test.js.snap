// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="home-main"
>
  <h1>
    Welcome to Calculator App!
  </h1>
  <h2>
    Let us Do some Math!
  </h2>
</div>
`;
